src/w30.rs

```diff
@@ -0,0 +1,26 @@
+//! # Handle W30 messages.
+
+use crate::context::Context;
+use crate::message::MsgId;
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+/// Status Update ID.
+#[derive(
+    Debug, Copy, Clone, Default, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize,
+)]
+pub struct StatusUpdateId(u32);
```

Comment:
if this is the `id` column from the db it should be `i64` rather than `u32`

Reply:
copied that from MsgId :)

Reply:
i suggest not to introduce new things in scope of this pr, we can revisit that at some point for all ids. i also doubt, it is a problem in practice in my lifetime ;)